mnist/README.md

```diff
@@ -14,10 +14,16 @@ Run the example with:
 python main.py
 ```
 
-By default the example runs on the CPU. To run on the GPU, use: 
+By default, the example runs on the CPU. To run on the GPU, use: 
 
 ```
 python main.py --gpu
 ```
 
 To run the PyTorch or Jax examples install the respective framework.
+
+To run the example with the Fashion-MNIST dataset, use:
+
+```
+python main.py --fashion_mnist
+```
```

Comment:
Could you just say "For a full list of options run `python main.py --help`" ? We want to keep the README minimal and general for this example.